### PR TITLE
remove unnecessary anonymous function

### PR DESCRIPTION
**Pull request by deads2k**
There is an upstream replacement for this function.  This cleans up the last usages.

blocks https://github.com/openshift/origin/pull/16861
/kind bug

### CODE REVIEW

File: test/integration/cli_get_token_test.go [hunk 1]
```diff
@@ -6,11 +6,11 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 	"github.com/openshift/origin/pkg/cmd/util/tokencmd"
 	userclient "github.com/openshift/origin/pkg/user/generated/internalclientset/typed/user/internalversion"
 	testutil "github.com/openshift/origin/test/util"
 	testserver "github.com/openshift/origin/test/util/server"
+	"k8s.io/client-go/rest"
```
Comment (mfojtik) [inline test/integration/cli_get_token_test.go]: move this up

File: test/integration/router/router_http_server.go [hunk 2]
```diff
@@ -19,6 +19,8 @@ import (
 	kapi "k8s.io/kubernetes/pkg/api"
 
 	"github.com/openshift/origin/pkg/cmd/util"
+
```
Comment (mfojtik) [inline test/integration/router/router_http_server.go]: nuke this new-line

**Comment by mfojtik** [comment]
minor nits, LGTM

**Comment by deads2k** [comment]
comments addressed.  tagging

**Pull request merged**